File: CarRental.Web/Models/AccountChangePasswordModel.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace CarRental.Web.Models
{
    public class AccountChangePasswordModel
    {
        public string LoginEmail { get; set; }
        public string OldPassword { get; set; }
        public string NewPassword { get; set; }
    }
}

File: CarRental.Web/Binding/Models/AccountChangePasswordModel.js

(function (cr) {
    var AccountChangePasswordModel = function () {

        var self = this;

        self.LoginEmail = ko.observable("");
        self.OldPassword = ko.observable("").extend({
            required: { message: "Old Password is required" },
            minLength: { message: "Old Password must be at least 6 characters", params: 6 }
        });
        self.NewPassword = ko.observable("").extend({
            required: { message: "New Password is required" },
            minLength: { message: "New Password must be at least 6 characters", params: 6 }
        });
    }
    cr.AccountChangePasswordModel = AccountChangePasswordModel;
}(window.CarRental));
